src/controllers/bus.js

```diff
@@ -0,0 +1,107 @@
+import successRes from '../helpers/successHandler';
+import errorRes from '../helpers/errorHandler';
+import { paginate } from 'paginate-info'
+import model from '../database/models';
+const {Bus}=model;
+export const createBus = async (req,res)=>{
+    try {
+        const bus= await Bus.create(req.body);
+        if(!bus){
+            return errorRes(res, 404, 'There was an error while creating a bus');
+        } else {
+            return successRes(res, 201, 'Bus created successfully', bus);
+        }
+        
+    } catch (error) {
+      return errorRes(res, 500,error);
+      
+    }
+}
+// export const getBus= async (req,res)=>{
+//     try {
+//         const bus=await Bus.findAll();
+//         return successRes(res, 200, 'Buses found', bus);
+
+//     } catch (error) {
+//         return res.status(400).send({
```

Comment:
It can be better if you remove commented codes

Reply:
Thanks, Issa I have removed those comments